app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from __future__ import annotations
+from app.players.player import Player
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+
+
+def calculate_team_total_rating(team: list[Player]) -> int:
+    return sum([player.get_rating() for player in team])
```

Comment:
no need to use list comprehension , you can use generator expression instead which is more efficient. simply remove []